### DISCUSSION TITLE
Pourquoi pas baisser la gloire perdue chaque semaine?

### DISCUSSION THREAD

**Discussion by aureochocob0n** [Crownicles/Crownicles#2488] [Nouvelles idées]
Bonjour à tous!

Aujourd'hui, je vous fais une proposition particulière : baisser le nombre de points de gloire perdu chaque semaine. 
Mais pourquoi cette suggestion ? Tout simplement parce que je me suis rendue  compte de quelque chose: plus on monte en points de gloire et plus on en perds.
 Mais c'est logique me diriez-vous, bien-sûr que ça l'est. Cependant, nous perdons TROP de gloire, personnellement je suis actuellement ligue élite et avec 1688 de gloire j'ai perdu 71 points. 
Et là, j'ai pleuré, parce que je n'ai pas l'occasion en une semaine de faire une vingtaine de fight, surtout si le tiers est prit par des gros levels où la limite de gloire est proche.. donc en somme, il faut être hyper actif et faire énormément de fight chaque semaine si je veux ne serait-ce que me maintenir.

Conclusion: je n'ai pas autant de temps à consacrer au jeu, et je doute être le seul, donc pourquoi nous obliger à être aussi actif sur les fight si c'est pour nous "dégoûter" du concept ?

Merci de m'avoir lu

**Comment by OrionYp** [comment]
Si on baisse le nombre de points perdus tout le monde va se retrouver en ligue légendaire facilement

**Comment by OrionYp** [comment]
La semaine dernière j'ai gagné 428 de gloire et j'en ai perdu que 87 ce matin à la compression, pour quelqu'un d'actif c'est même très simple de monter en gloire

**Comment by Carsebel** [comment]
C'est pas toutes les semaines que tu vas gagner 428 pts de gloire. Tu reviens après une suspension donc normal que tu gagnes pas mal.

**Comment by Procion06** [comment]
où comment prendre un exemple exceptionnel pour en faire une généralité xD

**Comment by Jajadu69** [comment]
La majorité des gens qui fight ne font pas 428 points par semaine. 
Surtout que dans certaines ligues il y a un gros manque d'adversaire. 
Tu es en or et tu prends -50 gloires ça fait mal

**Reply by Cyrpa** [reply]
Et c'est très bien comme ça, sinon tout le monde serait en ligue légende depuis longtemps et les fights seraient déséquilibrés et prévisibles. Il faut que la ligue continue de représenter un minimum le niveau des joueurs.

**Reply by aureochocob0n** [reply]
Cyrpa, je ne dis pas qu'il faut supprimer la perte de gloire, seulement de baisser la gloire perdue. Puisqu'à partir du moment où l'on ne peut pas faire plus d'une dizaine de fight à la semaine, ce qui est compréhensible, nous avons de grandes chances de stagner voir, comme dans mon cas perdre de la gloire. 
De plus, les ligues représenteront toujours la différence de niveau, avec le nombre de joueurs actifs dans les combats il est impossible de monter haut sans battre des joueurs hauts levels et donc d'être soit même "hauts level"

**Reply by Jajadu69** [reply]
C'est exactement ce que j'allais dire 😁

**Reply by Carsebel** [reply]
Toutes les semaines ne sont pas pareilles. Tu as perdu plus de gloire que tu en as gagné mais la semaine prochaine ça peut être l'inverse. 
J'ai perdu 102 de gloire cette semaine et j'en ai pas gagné autant en retour, au contraire. 
Et tu peux faire plus qu'une dizaine de fights par semaine. Tu vas chercher les personnes avec +/- de 450 de gloire de différence avec toi donc tu as de quoi faire

**Reply by Cyrpa** [reply]
+ si on baisse la gloire perdue les écarts se creuseront d'avantage donc le manque d'adversaires que jaja soulignait s'en trouvera amplifié, t'auras des gens avec 2700 de gloire, des gros trous partout, personne pour les combler...

**Reply by aureochocob0n** [reply]
Carsebel, je ne parlais pas de la dizaine de fight comme si je n'avais que 3 adversaires à affronter, j'en ai bien plus que ça. Mais est-ce que j'ai le temps d'en faire plus des fight? La réponse est non

**Comment by aureochocob0n** [comment]
Nous pourions aussi prendre une autre position: perdre de la gloire en fonction de celle qu'on a + du nombre de fight gagnés en 1 semaine.
 De cette manière, si on gagne 10 combats on perd un peux plus que celui qui en a fait 5 même si on a la même gloire. Cela pourrait également dissuader de faire des fight pour gagner du +2 et donc de faire perdre du temps précieux à ceux qui ne peuvent pas faire énormément de fight

**Reply by Cyrpa** [reply]
Résultat de cette position : 
Les joueurs actifs seront plus pénalisés que ceux qui ne font rien, alors que leur activité sera la seule chose qui les différencie = injustice, baisse d'activité générale puisque même résultat au bout du compte. 

Chacun peut faire autant de fight qu'il le souhaite, si ton temps est si précieux tu peux l'investir autrement, mais ça paraît osé de venir te plaindre de pas monter en ligue alors que tu fight pas beaucoup. Si tu veux monter, sois actif ?
Le système actuel de compression a l'air stable et équilibré (de mon point de vue), aucune raison de le changer.

**Reply by Carsebel** [reply]
Ça me tue de le dire, mais t'as raison. 
Sur ce, je vais mourir en paix 😅

**Reply by aureochocob0n** [reply]
Pas la peine d'être agressif, tu es désagréable cyrpa. De plus, encore une fois tu n'interprètes que ce qui t'arrange.
À AUCUN moment je ne dis que le résultat sera le même peux importe le nombre de fight, cette phrase que tu as prononcé n'a aucun sens puisque sinon tout le monde aurait la même gloire. 
Seulement, mon idée est de "limiter" la perte de gloire lors des semaines où nous n'avons pas le temps nécessaire au maintien.
Enfin, je ne souhaite pas que ce système permette aux "moins actifs" de monter plus facilement en gloire que les joueurs actifs. 
Comme ils auront fais moins de fight bien sûr qu'ils seront plus impactés par la baisse, mais celle-ci est tellement énorme qu'elle décourage de nombreux joueurs à profiter de cet aspect du jeu, qui pourtant représente une grande partie de celui-ci ce qui est vraiment dommage. 

Je ne comprends pas ton argument visant la baisse d'activité des joueurs, je m'explique. Premièrement, avec le système actuel comme je l'ai énoncé plus haut il <décourage de nombreux joueurs à profiter de cet aspect du jeu>. Des joueurs qui n'ont pas la possibilité d'être hyper actifs sur les combats.
Ainsi, un joueur qui gagne plus de combats qu'un autre aura naturellement une gloire plus élevés, car même s'il perd un peux plus que celui qui a gagné moins il sera toujours devant. 

Ce système dissuaderait les fight où l'on gagne du +2 sans toutefois pénaliser les joueurs actifs, puisqu'encore une fois, le joueur qui a gagné plus de fight perdrait seulement "un petit peux plus" que l'autre.

**Reply by Cyrpa** [reply]
Mais pour quoi faire ? 
La compression est née du refus de perdre des points de gloire dans les fights directement, évidemment qu'elle est importante, si tu veux te maintenir tu dois fight assez souvent (et encore, pas tant que ça au vu de ta ligue). C'est ça ou perdre une grande partie de ta gloire à cause de malchance par exemple.
Ensuite, je sais pas vraiment de quels joueurs "découragés" par la perte de gloire tu parles, simplement, si tu montes, tu vas fight que des lvl 110 et là t'auras de bonnes raisons d'être découragé 👍, une fois que t'es dans une ligue à ton niveau c'est pas toujours une bonne idée de monter plus haut.
Et encore une fois, c'est pas la faute de draftbot si des fois t'as pas le temps de jouer, je vois pas pourquoi t'aurais le droit de gagner de la gloire pour cette raison (oui car ne pas perdre autant que tu dois perdre = gagner).
Enfin, je pense aux joueurs dans le haut du top gloire. Les fights gagnés ne peuvent pas rapporter 30 de gloire, donc on a besoin d'en gagner beaucoup. Si tu bases la compression sur le nombre de fights gagnés, c'est nous qui prenons (et pour ma part je continuerai toujours à fight pour du +2, voili voilou).

**Reply by aureochocob0n** [reply]
Enfait, on s'est mal compris. Je suis pour que les joueurs normalement actifs ne se fassent pas découragés par la compression. Dans tous les cas, comme je l'ai déjà dis ils ne pourront plus monter à un certain stade car les autres joueurs seront trop hauts levels pour eux. Mais dans ce cas, il ne pourront pas monter car ils ne sont pas assez puissants ce qui ne va pas les décourager, au contraire, ils voudront jouer encore plus pour augmenter leurs stats. 
Mais là, l'ennemi des joueurs actuels n'est pas leurs adversaires mais la terrible compression qui, à basse ligue soit là où se trouvent la majorités d'entre eux les empêche considérablement de pouvoir progresser

**Reply by aureochocob0n** [reply]
Je n'ai pas dit que ma proposition en fonction des fight gagnés était la solution, seulement, moi j'essaie d'en trouver des solutions pour éviter justement ce "j'en ai marre, ça sert à rien de combattre de toute façon je perds plus que se que je gagne"

**Reply by Cyrpa** [reply]
Je comprends tes arguments, mais si certains perdent plus qu'ils ne gagnent et sont actifs, c'est juste qu'ils sont pas assez forts, si on restreint la compression ils vont monter et se heurter à un mur de défaites qui sera bien plus décourageant que la compression puisque les points qu'ils gagneront contre les joueurs qu'ils battent d'habitude diminueront (puisqu'eux sont montés). Donc, diminution des points gagnés, donc descente obligatoire dans leur ligue d'origine. Retour au point de départ. 
La stabilité actuelle vaut mieux qu'un découragement total des fights. (Car oui, si tu perds 80-90% de tes fights ça decourage)

**Reply by aureochocob0n** [reply]
Je comprends

**Comment by Carsebel** [comment]
Ça démoralise bcp de perdre autant de gloire mais à force tu t'y habitues. 
Quand tu perds 100pts la 1ere fois ça fout un coup et par la suite tu ne fais plus attention.

**Comment by Gabrielre14** [comment]
Aureo, cela doit faire 3 mois que je me maintiens entre 2000 et 2150 de gloire sans aucun réel problème. Et cela même actuellement en chevalier, qui n'est pas une classe facile.

Si tu as l'impression de perdre plus de gloire que tu n'en gagnes, c'est peut-être juste que ta ligue est trop élevée par rapport à ton niveau. Le système de combat récompense certes les plus actifs, mais le niveau est aussi nécessaire pour atteindre les meilleures ligues. 

Je ne pense pas que le nerf de gloire soit trop important. Avec une victoire par jour (ce qui n'est pas vraiment difficile), on gagne environ... 140 points de gloire ! Largement assez pour compenser le nerf de fin de saison et progresser.

**Comment by OrionYp** [comment]
Sinon je propose autre chose, remettre la gloire de tout le monde à 0 chaque semaine et diminuer les écarts de points par ligue

**Reply by Cyrpa** [reply]
Refusé.

**Reply by Gabrielre14** [reply]
Le problème de ce système, ce qui voulait être évité avec le système d'écart de gloire, c'est que des débutants se fassent écraser quotidiennement par des joueurs bien meilleurs. 
Avec le système actuel, les bons joueurs actifs ne combattent jamais les débutants (sauf cas particuliers, un ancien qui rejoue ou un joueur aventure qui commence le PvP). Un reset hebdomadaire, ou même mensuel troublerait cet objectif de créer des groupes de combat par niveau.

**Reply by OrionYp** [reply]
Pfff on peut même plus rire tranquille
